services/QuillLMS/app/models/concept_feedback.rb

```diff
@@ -37,6 +41,31 @@ def as_json(options=nil)
     data
   end
 
+  def queue_translation
```

Comment:
I think `queue` here is an implementation detail and probably not needed for the method name.  Instead, something like `translate_description` or `map_description_translation`.